yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/yb-voyager-pg-gather-assessment-metadata.sh

```diff
@@ -193,7 +193,7 @@ WHERE schemaname = ANY(ARRAY[string_to_array('$schema_list', '|')])
     SELECT 1 
     FROM pg_stat_all_tables pgst2 
     WHERE pgst2.schemaname = pgst1.schemaname 
-      AND pgst2.last_analyze IS NOT NULL
+      AND (pgst2.last_analyze IS NOT NULL OR pgst2.last_autoanalyze IS NOT NULL)
```

Comment:
This seems to address the scenario where auto analyze was switched on; and there was no explicit analyze run. 

mihnea's doc has an additional item. Are we also planning to tackle this? 
"PG runs ANALYZE by default so I was surprised as this should basically never be a problem. In my example one table is very small (under 50 rows) so it was not auto-analyzed based on PG rules (pg auto-analyze default thresholds are 50 rows and 10% new changes). So I'd guess Voyager was complaining based on that.
I'd say that, if a use-case with hundreds of tables has 1-2 tiny tables that may not have stats, that is not so critical. I'd suggest we can just assume those tables are small and move on. We could also look at table sizes with pg_table_size or so to confirm."

Reply:
This is something we already covered. If you remember, we discussed this, and this is what we report. 
This is what we report

1. no tables analysed at all in schema - Yes report 
2. All tables analysed in schema - No
3. 1 table analysed in schema - No

The problem in his case was that the auto-analyze wasn't handled, and hence it was considering the first case where nothing was analyzed, and hence reporting it. But now this should also not be reported